src/index.html

```diff
@@ -30,11 +30,15 @@ <h3>par John Allsopp, le 7 avril 2000</h3>
 			<p>[…] La notion de flexibilité dont j'ai parlé jusqu'ici fait référence à « l'adaptabilité ». Tout ce que j'ai dit jusqu'à présent pourrait se résumer à ceci : construisez des pages que l'on peut adapter. Construisez des pages qui sont accessibles, indifféremment de la plateforme ou de l'écran que votre lecteur a choisi ou qu'il doit utiliser pour parvenir jusqu'à vos pages. Cela signifie faire des pages qui sont lisibles indépendamment de la résolution ou de la taille de l'écran, du nombre de couleurs (et rappelez-vous aussi que certaines pages doivent être imprimées ou lues par des logiciels de synthèse vocale, ou encore lues via des navigateurs braille). Cela signifie des pages qui s'adaptent aux besoins d'un lecteur dont la vue est tout sauf parfaite et qui désire lire des pages avec des polices de caractères de très grande taille. […]</p>
 		</article>
 		<aside>
+      <img src="./img/airplane.svg" alt="" height="50px" width="50px" />
```

Comment:
in `height` and `width` attributes of `img` unit should be missing (implicit `px`) or in percent.
```suggestion
      <img src="./img/airplane.svg" alt="" height="50" width="50" />
```